test/cmdlineTests/constant_optimizer_yul/output

```diff
@@ -6,23 +6,42 @@ Optimized IR:
  *                !USE AT YOUR OWN RISK!               *
  *=====================================================*/
 
+/** @use-src 0:"constant_optimizer_yul/input.sol", 1:"#utility.yul" */
 object "C_12" {
     code {
         {
+            /** @src 0:61:418 */
             mstore(64, 128)
-            if callvalue() { revert(0, 0) }
-            sstore(0, shl(180, 1))
-            let _1 := datasize("C_12_deployed")
+            if /** @src 0:61:418 */
+            callvalue()
+            /** @src 0:61:418 */
```

Comment:
Can you please check with @christianparpart that we have a test that a comment at this point is properly applied to the if body?

Reply:
is definitely supported (by looking at Scanner.cpp i at least don't see why not) :)